lib/suspenders/generators/app_generator.rb

```diff
@@ -31,11 +31,8 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :skip_system_test,
       type: :boolean, default: true, desc: "Skip system test files"
 
-    class_option :skip_turbolinks,
-      type: :boolean, default: true, desc: "Skip turbolinks gem"
-
-    class_option :skip_spring, type: :boolean, default: true,
-      desc: class_options[:skip_spring].description
+    class_option :css,
+      type: :string, default: "postcss", aliases: "-c", desc: "Choose CSS processor"
```

Comment:
I wonder if we need to update the description to load all options, similar to what we do with the database option.


https://github.com/thoughtbot/suspenders/blob/5624ca5903f2d1cf13074de668524a4ef05a7780/lib/suspenders/generators/app_generator.rb#L10-L11

Reply:
I also wonder if we should test the build by passing in an unsupported option.


```
suspenders some_app --css=no_supported
```

Reply:
We tested this with `rails new` and Rails will throw an error.